client/components/tinymce/plugins/wpcom-view/views/simple-payments/style.scss

```diff
@@ -119,7 +119,13 @@
 	top: -3px;
 	width: 43px;
 	perspective-origin: 21.125px 6px;
-	font: normal normal 500 normal 10px / normal "Helvetica Neue",Helvetica, Arial, sans-serif;
+	font-family: "Helvetica Neue",Helvetica, Arial, sans-serif;
+	font-size: 10px;
+	font-stretch: normal;
+	font-style: normal;
+	font-variant: normal;
+	font-weight: 500;
+	line-height: normal;
```

Comment:
I'll keep `line-height` and `font-weight` but I think it's safe to remove `font-variant` and `font-stretch` as we rarely see them being used.